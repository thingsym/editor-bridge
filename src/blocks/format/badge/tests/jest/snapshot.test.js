/**
* External dependencies
*/
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

/**
 * Internal dependencies
 */
import {
	default as InlineBadgeUI,
} from '../../badge-popover.js';

describe( 'InlineBadgeUI snapshot', () => {
	it( 'InlineBadgeUI snapshot', () => {
		const elem = shallowToJson( shallow( <InlineBadgeUI /> ) );
		expect( elem ).toMatchSnapshot();
	} );
} );
