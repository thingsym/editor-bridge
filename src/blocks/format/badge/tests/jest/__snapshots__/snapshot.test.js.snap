// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineBadgeUI snapshot InlineBadgeUI snapshot 1`] = `
<BadgePopoverAtLink
  className="components-inline-badge-popover"
>
  <TabPanel
    tabs={
      [
        {
          "name": "backgroundColor",
          "title": "Background",
        },
        {
          "name": "color",
          "title": "Text",
        },
        {
          "name": "style",
          "title": "Style",
        },
      ]
    }
  >
    <Component />
  </TabPanel>
</BadgePopoverAtLink>
`;
